### PR TITLE
Refactor db_read.py to replace urlopen with requests.

### PR DESCRIPTION
**Pull request by dherbst**
Related #2852 

### Technical

Refactor db_read.py to replace urlopen with requests.   

### Testing

```python
from openlibrary.catalog.importer.db_read import *
read_from_url(api_get()+'/works/OL86707W')
```
```json
{'description': "Stellaluna is a baby bat, but she lives in a bird's nest with a bird family. How did she get there? Will she ever find her bat family? --back cover", 'links': [{'url': 'https://en.wikipedia.org/wiki/Stellaluna', 'title': 'Stellaluna - Wikipedia', 'type': {'key': '/type/link'}}, {'title': 'Stellaluna - Storyline Online', 'url': 'https://www.storylineonline.net/books/stellaluna/', 'type': {'key': '/type/link'}}, {'url': 'https://www.rom.on.ca/en/rom-at-home/rom-storytime/stellaluna-by-janell-cannon', 'title': 'ROM Storytime: “Stellaluna” by Janell Cannon | Royal Ontario ...', 'type': {'key': '/type/link'}}], 'title': 'Stellaluna', 'covers': [10515731, 10500690], 'first_publish_date': '1990', 'subject_people': ['Stellaluna', 'Pip', 'Flitter', 'Flap', 'Mother Bat'], 'key': '/works/OL86707W', 'authors': [{'author': {'key': '/authors/OL31906A'}, 'type': {'key': '/type/author_role'}}], 'excerpts': [{'comment': 'first sentence', 'author': {'key': '/people/seabelis'}, 'excerpt': 'In a warm and sultry forest far, far away, there once lived a mother fruit bat and her new baby.'}], 'type': {'key': '/type/work'}, 'subjects': ['owls', 'Juvenile Fiction', 'Aves', 'Bats', 'Birds', 'Ficción juvenil', 'Fiction', 'Juvenile software', 'Murciélagos', 'Software', 'Specimens', 'Toy and movable books', 'Picture books', 'Friendship', 'Board books', 'Spanish: Kindergarten', 'Short Stories', "Children's 4-8 - Picturebooks", "Children's Books/Ages 4-8 Fiction", 'Spanish: Grades 2-3', 'Spanish: Grades 1-2', 'Animals - Mammals', 'Animals - Birds', "Children's fiction", 'Bats, fiction', 'Birds, fiction', 'Friendship, fiction'], 'latest_revision': 31, 'revision': 31, 'created': {'type': '/type/datetime', 'value': '2009-10-17T11:24:20.769455'}, 'last_modified': {'type': '/type/datetime', 'value': '2021-01-25T19:16:51.709885'}}
```

### Screenshot

N/A

### Stakeholders

@cclauss

### CODE REVIEW

File: openlibrary/catalog/importer/db_read.py [hunk 1]
```diff
@@ -3,60 +3,68 @@
...
 
 staging = False
 
 db = web.database(dbn='postgres', db='marc_index', host='ol-db')
 db.printing = False
 
-def find_author(name): # unused
-    iter = web.query('select key from thing, author_str where thing_id = id and key_id = 1 and value = $name', {'name': name })
+
+def find_author(name):  # unused
```
Comment (cclauss) [inline openlibrary/catalog/importer/db_read.py:17]: Please mark this function with [`@deprecated`](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/marc/fast_parse.py#L16) to mark it for removal in the future.

File: openlibrary/catalog/importer/db_read.py [hunk 2]
```diff
@@ -3,60 +3,68 @@
...
     return "https://openlibrary.org%s/api/" % (':8080' if staging else '')
 
+
 def api_versions(): return api_url() + "versions?"
 def api_things(): return api_url() + "things?"
 def api_get(): return api_url() + "get?key="
 
-def get_versions(q): # unused
+
+def get_versions(q):  # unused
```
Comment (cclauss) [inline openlibrary/catalog/importer/db_read.py:58]: @deprecated please.

**Review by cclauss — CHANGES_REQUESTED** [review]
Please tag unused functions with @deprecated.  https://deprecated.readthedocs.io

File: openlibrary/catalog/importer/db_read.py (hunk 1 above)
Comment (dherbst) [reply openlibrary/catalog/importer/db_read.py:18]: Done.  I tried to test this and web does not have a query attribute, so find_author does not work.

**Review by cclauss — APPROVED** [review]
LTGM.  Thanks!

**Pull request merged**